jax/interpreters/xla.py

```diff
@@ -40,6 +40,9 @@
 from . import ad
 from . import masking
 
+xe = xc._xla
```

Comment:
Use lib.xe instead? (And below)

Reply:
Left this here because I removed it from the lib.